### PR TITLE
Add twitter to socials in footer

### PR DESCRIPTION
**Pull request by tupui**
Maybe we can also add Slack, Discord (but discord would be the Scientific Python one) and discuss? WDYT?

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -84,6 +84,8 @@ languages:
       socialmedia:
         - link: https://github.com/scipy/scipy
           icon: github
+        - link: https://twitter.com/SciPy_team
+        icon: twitter
```
Comment (tupui) [inline config.yaml:88]:
```suggestion
           icon: twitter
```

File: config.yaml [hunk 2]
```diff
@@ -84,6 +84,8 @@ languages:
       socialmedia:
         - link: https://github.com/scipy/scipy
           icon: github
+        - link: https://twitter.com/SciPy_team
+           icon: twitter
```
Comment (tupui) [inline config.yaml:88]:
```suggestion
          icon: twitter
```

**Review by rgommers — APPROVED** [review]
Thanks @tupui, Twitter icon is nice to have.

> Maybe we can also add Slack, Discord (but discord would be the Scientific Python one) and discuss? WDYT?

Slack is a private channel so there's nowhere to link to. Discord isn't part of the SciPy project; now that after all these years the site is about SciPy the project and nothing else, I'd prefer to keep it that way. So an icon doesn't seem to fit. If we do want to link out to a representative set if relevant community sites/projects, we should probably add a section under Community? It can also point to the Ecosystem part of numpy.org (best to leave it there and not repeat that kind of info on scipy.org)

**Comment by tupui** [comment]
Ok yes I will think about adding a section on the community page.

**Pull request merged**